deploy/docs/Non_Helm_Installation.md

```diff
@@ -0,0 +1,493 @@
+
+#### Deploy Fluentd
+
+In this step you will deploy Fluentd using a Sumo-provided .yaml manifest. This step also creates Kubernetes secrets for the HTTP Sources created in the previous step.
+
+Run the following command to create namespace `sumologic`
+
+```sh
+kubectl create namespace sumologic
+```
+
+Run the following command to create a Kubernetes secret containing the 9 HTTP source URLs previously created.
+
+```sh
+kubectl -n sumologic create secret generic sumologic \
+  --from-literal=endpoint-metrics=$ENDPOINT_METRICS \
+  --from-literal=endpoint-metrics-apiserver=$ENDPOINT_METRICS_APISERVER \
+  --from-literal=endpoint-metrics-kube-controller-manager=$ENDPOINT_METRICS_KUBE_CONTROLLER_MANAGER \
+  --from-literal=endpoint-metrics-kube-scheduler=$ENDPOINT_METRICS_KUBE_SCHEDULER \
+  --from-literal=endpoint-metrics-kube-state=$ENDPOINT_METRICS_KUBE_STATE \
+  --from-literal=endpoint-metrics-kubelet=$ENDPOINT_METRICS_KUBELET \
+  --from-literal=endpoint-metrics-node-exporter=$ENDPOINT_METRICS_NODE_EXPORTER \
+  --from-literal=endpoint-logs=$ENDPOINT_LOGS \
+  --from-literal=endpoint-events=$ENDPOINT_EVENTS
+```
+
+##### Use default configuration
+
+If you don't need to customize the configuration apply the `fluentd-sumologic.yaml` manifest with the following command:
+
+```sh
+curl https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/master/deploy/kubernetes/fluentd-sumologic.yaml.tmpl | \
+sed 's/\$NAMESPACE'"/sumologic/g" | \
+kubectl -n sumologic apply -f -
+```
+
+##### Customize configuration
+
+If you need to customize the configuration there are two commands to run. First, get the `fluentd-sumologic.yaml` manifest with following command:
+
+```sh
+curl https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/master/deploy/kubernetes/fluentd-sumologic.yaml.tmpl | \
+sed 's/\$NAMESPACE'"/sumologic/g" >> fluentd-sumologic.yaml
+```
+
+Next, customize the provided YAML file. Our [plugin](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/master/fluent-plugin-events/README.md#fluent-plugin-events) allows you to configure fields for events. Once done run the following command to apply the `fluentd-sumologic.yaml` manifest.
+
+```sh
+kubectl -n sumologic apply -f fluentd-sumologic.yaml
+```
+
+The manifest will create the Kubernetes resources required by Fluentd.
+
+
+#### Verify the pods are running
+
+```sh
+kubectl -n sumologic get pod
+```
+
+## Configure Prometheus
+
+In this step, you will configure the Prometheus server to write metrics to Fluentd.
+
+Install Helm:
+
+*Note the following steps are one way to install Helm, but in order to ensure property security, please be sure to review the [Helm documentation.](https://helm.sh/docs/using_helm/#securing-your-helm-installation)*
+
+Download Helm to generate the yaml files necessary to deploy by running
+
+```bash
+brew install kubernetes-helm
```

Comment:
We are missing the RBAC steps and helm init. Also should note this is only required if helm is not installed already.

Reply:
RBAC steps and helm init steps shouldn't be needed since these are the non-helm installation steps, correct?